src/js/big-play-button.js

```diff
@@ -34,7 +34,22 @@ class BigPlayButton extends Button {
    * @listens click
    */
   handleClick(event) {
-    this.player_.play();
+    const playPromise = this.player_.play();
+
+    const cb = this.player_.getChild('controlBar');
+    const playToggle = cb && cb.getChild('playToggle');
+
+    if (!playToggle) {
+      return;
```

Comment:
If there isn't a playToggle control, focus ought to move to either the player's `<div>`, or the `<video>` element, rather than just falling into a black hole! I guess the `<video>` element won't be there if the player uses a Flash tech, so the `<div>` makes sense.